core/arch/arm/plat-poplar/main.c

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2017, Linaro Limited
+ * All rights reserved.
```

Comment:
We don't the "All rights reserved" line any longer, please remove it in the other files added with Linaro copyright.

Reply:
@vchong FYI the new header format is documented in PR#1992
https://github.com/OP-TEE/optee_os/pull/1992/commits/453cae1374c98c2749899165715494f0f9f98a81

Reply:
Removed in all related files.